arena/src/main/java/ch/bfh/zodnrk/arena/model/Hero.java

```diff
@@ -0,0 +1,60 @@
+package ch.bfh.zodnrk.arena.model;
+
+import org.springframework.hateoas.RepresentationModel;
+
+public class Hero extends RepresentationModel<Hero> {
```

Comment:
Why do we need to duplicate the Hero and Party class in the Arena Module?

Reply:
Hero and Party classes are in all module part of it.